zerver/lib/actions.py

```diff
@@ -5065,6 +5065,14 @@ def do_change_notification_settings(
 
     send_event(user_profile.realm, event, [user_profile.id])
 
+    event = {
```

Comment:
I renamed these local variables to `legacy_event`, just for readability reasons, and also to avoid having the `event` name have two types in these functions.